src/conf.py

```diff
@@ -71,84 +102,78 @@ def multiprocess_log(filename, name=__name__):
         log.addHandler(_handler)
     return log
 
-#
-#
-#
-
-SRC_DIR = os.path.dirname(inspect.getfile(inspect.currentframe())) # ll: /path/to/adaptor/src/
-PROJECT_DIR = os.path.dirname(SRC_DIR)  # ll: /path/to/adaptor/
-
-CFG_NAME = 'app.cfg'
-DYNCONFIG = configparser.SafeConfigParser(**{
-    'allow_no_value': True,
-    # these can be used like template variables
-    # https://docs.python.org/2/library/configparser.html
-    'defaults': {'dir': PROJECT_DIR}})
-DYNCONFIG.read(join(PROJECT_DIR, CFG_NAME)) # ll: /path/to/lax/app.cfg
-
-def cfg(path, default=0xDEADBEEF):
-    lu = {'True': True, 'true': True, 'False': False, 'false': False} # cast any obvious booleans
-    try:
-        val = DYNCONFIG.get(*path.split('.'))
-        return lu.get(val, val)
-    except (configparser.NoOptionError, configparser.NoSectionError): # given key in section hasn't been defined
-        if default == 0xDEADBEEF:
-            raise ValueError("no value/section set for setting at %r" % path)
-        return default
-    except Exception:
-        raise
+LOG_DIR = PROJECT_DIR
+# TODO: re-enable once changes to lax-formula are deployed
+# if ENV != DEV:
+#    LOG_DIR = '/var/log/bot-lax-adaptor/'
+utils.writable_dir(LOG_DIR)
 
 #
 #
 #
 
-ENV = cfg('general.env')
-
 PATH_TO_LAX = cfg('lax.location')
 
-CACHE_PATH = cfg('lax.cache_path')
+# TODO: re-enable once changes to lax-formula are deployed
+CACHE_PATH = join(PROJECT_DIR, 'cache')
+# if ENV != DEV:
+#    CACHE_PATH = cfg('general.cache_path', CACHE_PATH)
+
+# TODO: remove once changes to lax-formula are deployed
+CACHE_PATH = cfg('lax.cache_path', CACHE_PATH)
```

Comment:
ah, I see what I was doing now. I've added `cache_path: /home/elife` back into the cfg file with a comment. we still need to stitch CI up with it's own app.cfg and a path to a shared cache file in the future, though.